.github/fabricbot.json

```diff
@@ -150,6 +119,10 @@
         "conditions": {
           "operator": "and",
           "operands": [
+            {
+              "name": "isPr",
+              "parameters": {}
+            },
```

Comment:
When this task is ported over, the configuration for Policy Service will need to be hand-edited. The `merged` action is not supported on Policy Service, so instead the configuration will need to check for `closed` with the `isMerged` condition.